docs/docs/setup/kubernetes.md

```diff
@@ -55,7 +56,7 @@ If you intend to use this feature, you'd have to set up and deploy PostgREST ser
 1. Setup PostgREST server
 
    ```bash
-    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/postgrest.yaml
+    kubectl apply -f https://raw.githubusercontent.com/ToolJet/ToolJet/main/deploy/kubernetes/deployment.yaml
    ```
```

Comment:
Hey @Abjcodes , the changes are not addressing issue https://github.com/ToolJet/ToolJet/issues/6302; kindly be sure to go through the issue before making the changes.

Reply:
You are right, the #6303 issue is to update the hyperlink of the text `here`, this PR is not addressing the requirements in the issue @Abjcodes , kindly revert it and make the change as mentioned in the issue.

Reply:
Hey @Abjcodes , were you able to progress with the requested changes? We can process this PR to review and merge as you address them.

Reply:
Hey @tejakummarikuntla, sorry for the delay, was busy for the past month. I will update the pr with the requested changes soon.

Reply:
Hey @Abjcodes , just a gentle reminder on this.